src/components/wizard/WizardStates.ts

```diff
@@ -4,16 +4,27 @@ const exclamationSmall = require('./assets/exclamationSmall.png');
 import {WizardStepsConfig} from './types';
 
 export const StatesConfig: WizardStepsConfig = {
-  enabled: {color: Colors.grey30, circleColor: Colors.grey60, enabled: true},
-  disabled: {color: Colors.grey50, circleColor: Colors.grey60},
-  error: {color: Colors.red30, icon: exclamationSmall, enabled: true, accessibilityInfo: 'Validation Error'},
-  skipped: {color: Colors.red30, enabled: true, accessibilityInfo: 'Not completed'},
+  enabled: {color: Colors.$textNeutralHeavy, circleColor: Colors.$outlineDisabled, enabled: true},
+  disabled: {color: Colors.$textDisabled, circleColor: Colors.outlineDefault},
```

Comment:
```suggestion
  disabled: {color: Colors.$textDisabled, circleColor: Colors.$outlineDisabled},
```

Reply:
Was it changed to grey60 so you're using the $outlineDefault token? anyways your missing the '$'

Reply:
There is a new design page for tokenized components: [Wizard](https://www.figma.com/file/HzaNWJziTYvLgU2vzQWjKg/DS-Tokenized?node-id=6457%3A400750) 
It's from there

Reply:
I see there both disabled and enabled borderline have the same token...

Reply:
Look at the bottom part, all the types are there.
You might have looked at `enabled` and `completed`.